grafana-tikv-dashboard.md

```diff
@@ -484,6 +484,47 @@ This section provides a detailed description of these key metrics on the **TiKV-
 - Initial Scanning Trigger Reason: The reason for triggering incremental scanning
 - Region Checkpoint Key Putting: The number of checkpoint operations logged to the PD
 
+### Snapshot restore 
+
+- Import CPU Utilization: CPU utilization aggregated by sst importer.
+- Import Thread Count: number of threads used by sst importer.
+- Import Errors: error counts during sst import.
+- Import RPC Duration: time spent for various RPC calls in sst importer.
+- Import RPC Ops: number of total RPC calls in sst importer.
+- Import RPC Count: number of inflight RPC calls in sst importer.
+- Import Write/Download RPC Duration: RPC time for write/download in sst importer.
+- Import Wait Duration: time spent on downloading task waiting in queue for execution.
+- Import Read SST Duration: time spent on reading from external storage of a file and download it to TiKV.
+- Import Rewrite SST Duration: time spent on rewriting SST based on rewrite rules. 
+- Import Ingest RPC Duration: time spent on sending ingest response in RPC call.
+- Import Ingest SST Duration: time spent on ingesting SST into RocksDB.
+- Import Ingest SST Bytes: number of bytes ingested.
+- Import Download SST Throughput: SST download throughput in bytes per second.
+- Import Local Write keys: ???
```

Comment:
anyone know what does local mean here?

Reply:
This is used by tidb-lightning, which send a stream of sorted kv pairs (instead of a reference to a remote SST), and TiKV will use the `SstFileWriter` of RocksDB to generate a SST.

The two metrics collects how many key / bytes it has written.

Reply:
I think we can skip this entry ?

Reply:
I see! thanks @YuJuncen 
yeah in that case it can be skipped